pkg/blocker/blocker.go

```diff
@@ -33,11 +39,17 @@ type Blocker struct {
 }
 
 func New(dis p2p.Blocklister, flagTimeout, blockDuration, wakeUpTime time.Duration, callback func(swarm.Address), logger logging.Logger) *Blocker {
+	if flagTimeout <= resolution {
+		panic(fmt.Errorf("flag timeout %v cannot be equal to or lower then the clock resolution %v", flagTimeout, resolution))
+	}
+	if wakeUpTime < resolution {
+		panic(fmt.Errorf("wakeup time %v cannot be lower then the clock resolution %v", wakeUpTime, resolution))
```

Comment:
wouldn't panicking here potentially cause inconsistencies in the store?

Reply:
These panics basically mean programmer error. It makes no sense to check for flagged peers more often than the sequencer resolution rate.

Reply:
right, but would panicking trigger the shutdown sequence? or would it just quick the process?

Reply:
Will crash the process.